jbpm-test-coverage/src/main/java/org/jbpm/test/jobexec/CheckCallCommand.java

```diff
@@ -0,0 +1,40 @@
+package org.jbpm.test.jobexec;
+
+import org.kie.api.executor.Command;
+import org.kie.api.executor.CommandContext;
+import org.kie.api.executor.ExecutionResults;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CheckCallCommand implements Command {
+    
+    private static final Logger logger = LoggerFactory.getLogger(CheckCallCommand.class);
+
+    private static Boolean commandExecuted = false;
+    private static Object LOCK = new Object();
```

Comment:
I believe that I always need some boolean var to determine if the command was really called (in assertion) because the waiting for the lock may timeout and without the boolean var I do not know if the lock was released by the command or timeout. I am not sure how CyclicBarrier would improve it.